dlme_airflow/task_groups/etl.py

```diff
@@ -19,13 +19,15 @@
 def etl_tasks(provider, task_group: TaskGroup, dag: DAG) -> TaskGroup:
     task_array = []
     source = catalog_for_provider(provider)
+    post_harvest = "add_thumbnail_urls"  # source.metadata.get('post_harvest', None)
```

Comment:
I think you forgot to remove the hard coded script?

Reply:
I did. I'm not suggesting this as "the change" but as a pattern to work towards.

Reply:
Ok, I misunderstood. I thought this was ready to merge.